lib/easy_meli/api_client.rb

```diff
@@ -49,9 +49,9 @@ def send_request(verb, path = '', params = {})
   end
 
   def check_authentication(response)
-    response_message = error_message_from_body(response.to_h)
+    response_message = error_message_from_body(response.to_h) unless response.parsed_response.is_a? Array
```

Comment:
I considered validating using:
```
    response_message = error_message_from_body(response.to_h) unless response.code == 200
```

Reply:
I would invert this and check to see if it is a Hash. You never know what else could be returned. I like the idea of checking the code too but I'm not 100% certain of all the codes that ML returns so opted not to. Once we have a history of logs we can look over them to get an idea of what is returned.